New browser install flow (#13902)
newrelic/docs-website: src/content/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring.mdx

## APM agents [#apm-agents]

> One way to [deploy browser monitoring](/docs/browser/new-relic-browser/getting-started/adding-apps-new-relic-browser) for an app is using a New Relic APM agent.

Reviewer: ```suggestion
One way to [deploy browser monitoring](/docs/browser/new-relic-browser/getting-started/adding-apps-new-relic-browser) for an app is using one of our APM agents.
```
